python/app/output_widget.py

```diff
@@ -131,6 +133,11 @@ def add_error(self, text):
 
         """
 
+        # if shotgun/toolkit is available, log the error message to the current
+        # engine.
+        if sgtk:
+            sgtk.platform.current_engine().logger.error(text)
```

Comment:
I suggest `sgtk.platform.current_engine().log_error(text)` for backwards comp.

Reply:
changed!